### DISCUSSION TITLE
设置页面又打不开了

### DISCUSSION THREAD

**Discussion by tjwwhaley** [shuax/MouseInc.Settings#480] [General]
设置页面又打不开了，你们呢？

昨天还好好的

**Comment by gbsix** [comment]
win7，离线界面包也打不开了。原因是：

Microsoft Edge and WebView2 ending support for Windows 7 and Windows 8/8.1

win10也不行：

tools.shuax.com 的响应时间过长。

**Comment by Walker0501** [comment]
> 设置页面又打不开了，你们呢？
> 
> 昨天还好好的

是不行了，不晓得现在是谁在维护

**Reply by gbsix** [reply]
安装[离线界面包](https://github.com/shuax/MouseInc.Settings/discussions/362)就可以打开了

**Reply by Walker0501** [reply]
嗯嗯，主要是OCR

**Reply by LihanChen2004** [reply]
> 安装[离线界面包](https://github.com/shuax/MouseInc.Settings/discussions/362)就可以打开了

呜呜呜谢谢 解决了

**Comment by HengXin233** [comment]
最开始设置和网站都进不了，挂上梯子后就能进了

**Reply by Walker0501** [reply]
确实可以，又可以耍了
